### DISCUSSION TITLE
openlrc: A python package to transcribe with whisper & translate with LLMs.

### DISCUSSION THREAD

**Discussion by zh-plus** [openai/whisper#2137] [Show and tell]
Excited to introduce openlrc: https://github.com/zh-plus/openlrc!

Open-Lyrics is a Python library designed for transcribing audio files swiftly using [faster-whisper](https://github.com/guillaumekln/faster-whisper), and then converting the transcribed text into .lrc/.srt files in any chosen language with the help of LLMs, such as [OpenAI-GPT](https://github.com/openai/openai-python) or [Anthropic-Claude](https://github.com/anthropics/anthropic-sdk-python).

### Highlights:
- Advanced audio preprocessing to minimize errors (includes Loudness Normalization and optional Noise Suppression).
- Context-sensitive translation enhancing accuracy. For more information, see our [prompt](https://github.com/zh-plus/openlrc/blob/master/openlrc/prompter.py).

**Comment by eguar11011** [comment]
Why use an LLM if Whisper performs translation directly?

**Reply by zh-plus** [reply]
Using LLM as an independent translation model results in better-quality translations.
